src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -564,9 +561,10 @@ void RenderTarget::initialize()
 void RenderTarget::applyCurrentView()
 {
     // Set the viewport
-    IntRect viewport = getViewport(m_view);
-    int     top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
-    glCheck(glViewport(viewport.left, top, viewport.width, viewport.height));
+    IntRect viewport(getViewport(m_view));
```

Comment:
```suggestion
    IntRect viewport = getViewport(m_view);
```
More unnecessary style changes.